### ISSUE TITLE
go get not working for the project due to changed dependencies in cloudfoundry/cli project

### ISSUE THREAD

**Issue by mehrotra-prateek** [bluemixgaragelondon/cf-blue-green-deploy#4]
I am trying to perform go get after forking the repository to fix #3 and am getting below errors due to change in cloudfoundry/cli project

```
package github.com/cloudfoundry/cli/cf/configuration/config_helpers: cannot find package "github.com/cloudfoundry/cli/cf/configuration/config_helpers" in any of:
        $GOROOT
        $GOPATH
package github.com/cloudfoundry/cli/cf/configuration/core_config: cannot find package "github.com/cloudfoundry/cli/cf/configuration/core_config" in any of:
     $GOROOT
     $GOPATH
package github.com/cloudfoundry/cli/plugin/fakes: cannot find package "github.com/cloudfoundry/cli/plugin/fakes" in any of:
        $GOROOT
        $GOPATH
```

the above erring dependencies should look like below now 

```
"github.com/cloudfoundry/cli/cf/configuration/confighelpers"
"github.com/cloudfoundry/cli/cf/configuration/coreconfig"
"github.com/cloudfoundry/cli/plugin/pluginfakes"
```
There will be fixes to getSpaceGuid function accordingly.

I have done these fixes in my fork but now found internationalization is also broken in main.go

**Comment by bluemixgaragelondon** [comment]
Unfortunately it's unlikely that this plugin will support `go get` any time soon.  One of the cloudfoundry/cli packages that we depend on has a dependency on generated internationalisation code that `go get` can't handle.

For this reason we've written `script/build` which calls `script/deps` - if you look in there you'll see we call `bin/generate-language-resources` from the cloudfoundry/cli project which generates the necessary code.  This solution isn't particularly elegant, but I think it's the simplest way to solve the problem at the moment.

If you build using those scripts, it builds against a fixed version of the cloudfoundry/cli project specified at the top of `script/deps`.  The version we had was rather old, so I've updated it to the latest stable release (v6.17.0).

Fixing the version of the dependency is not strictly in line with the go philosophy, but hasn't caused us any issues and has helped to isolate us from the problems you encountered (we were previously using godep to manage this, but found that it added complexity without any benefit).

So in summary, I would try working with the latest HEAD, and recommend using the scripts in the `script` directory for building and testing.  Please let us know if you have any further issues.

**Comment by bluemixgaragelondon** [comment]
I'm going to close this issue for now, please re-open if the above suggestion doesn't work.
